tests/features/common/actions/api.actions.js

```diff
@@ -0,0 +1,258 @@
+import { mainHttpClient } from '../../../../src/httpClient'
+import { expect } from 'chai'
+import { v4 as uuidv4 } from 'uuid'
+
+const REACT_APP_MLRUN_API_URL = 'http://localhost:3000'
+
+const newJobTemplate = {
+  task: {
+    spec: {
+      parameters: {},
+      inputs: {},
+      hyperparams: {},
+      secret_sources: [],
+      param_file: '',
+      tuning_strategy: 'list',
+      function: 'hub://aggregate',
+      handler: 'aggregate',
+      input_path: '',
+      output_path: 'v3io:///projects/{{run.project}}/artifacts/{{run.uid}}'
+    },
+    metadata: {
+      labels: {}
+    }
+  },
+  function: {
+    metadata: {},
+    spec: {}
+  }
+}
+
+const action = {
+  deleteAPIMLProject: async function(
+    driver,
+    mlProjectName,
+    expectedStatusCode
+  ) {
+    await driver.sleep(1000)
+    await mainHttpClient
+      .delete(`${REACT_APP_MLRUN_API_URL}/api/projects/${mlProjectName}`)
+      .then(res => {
+        expect(res.status).equal(expectedStatusCode)
+      })
+  },
+  deleteAPIFeatureSet: async function(
+    projectName,
+    featureSetName,
+    expectedStatusCode
+  ) {
+    await mainHttpClient
+      .delete(
+        `${REACT_APP_MLRUN_API_URL}/api/projects/${projectName}/feature-sets/${featureSetName}`
+      )
+      .then(res => {
+        expect(res.status).equal(expectedStatusCode)
```

Comment:
expect is a method from the UI testing library "chai", you have import in the second line of the file. I don't think you should use it here. Please use vanilla JS or lodash library.